src/flyte/_trigger.py

```
@@ -722,6 +722,8 @@ def __post_init__(self):
             raise ValueError("Trigger name cannot be empty")
         if self.automation is None:
             raise ValueError("Automation cannot be None")
+        if self.description and len(self.description) > 255:
+            object.__setattr__(self, "description", self.description[:255])
```

We truncated at 255 characters as that is the limit of backend db.

---

Should we raise an error or a warning if it exceeds the limit? @wild-endeavor